stripe/raw_request.py

```diff
@@ -0,0 +1,49 @@
+from stripe import api_requestor, util
+from stripe.api_version import _ApiVersion
+
+
+def _raw_request(method_, url_, **params):
+    params = None if params is None else params.copy()
+    api_key = util.read_special_variable(params, "api_key", None)
+    idempotency_key = util.read_special_variable(
+        params, "idempotency_key", None
+    )
+    client = util.read_special_variable(params, "client", None)  # for testing
```

Comment:
did we end up using this for testing?

Reply:
Yes, we set it to `http_client` for tests in `test_raw_request`: https://github.com/stripe/stripe-python/pull/965/files#diff-43ede50a826bcdfb9c8f2275c1731c87383b93ad972822a6b281ba8001910aa0R44